pom.xml

```diff
@@ -75,6 +75,24 @@
         <bootclasspath.compile>${bootclasspath.java6}</bootclasspath.compile>
         <bootclasspath.testCompile>${bootclasspath.java6}</bootclasspath.testCompile>
         <antlr.testinprocess>true</antlr.testinprocess>
+        <maven-bundle-plugin.version>2.3.7</maven-bundle-plugin.version>
```

Comment:
Of course I can inline the version number and I will test with a more current version of the bundle plugin.